python/test/ngraph/test_ops_unary.py

```diff
@@ -0,0 +1,58 @@
+# ******************************************************************************
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+import numpy as np
+import pytest
+
+import ngraph as ng
+
+
+def _get_runtime():
+    manager_name = pytest.config.getoption('backend', default='CPU')
+    return ng.runtime(manager_name=manager_name)
+
+
+def _run_unary_op_node(input_data, unary_op, as_node=True):
```

Comment:
If you use `as_node` you don't need to create or pass the parameter.
Let's split this into 2 functions, one to run the operation on a node, the other on numeric data.
You can then run both in each test, so no need to pass the extra `as_node` param.

Reply:
updated